website/docs/cloud-docs/vcs/github.mdx

```diff
@@ -90,23 +90,31 @@ Leave the page open in a browser tab. In the next step you will copy values from
 
 1. Click the green "Authorize `<GITHUB USER>`" button at the bottom of the authorization page. GitHub might request your password to confirm the operation.
 
```

Comment:
@robin-norwood I plan to make these same changes to the following files in addition to this one: github-enterprise.mdx, gitlab-com.mdx, bitbucket-cloud.mdx, azure-devops-services.mdx. For the following: gitlab-eece.mdx, bitbucket-server.mdx, azure-devops-server.mdx the SSH keypair section is slightly different so there will be some nuances between the changes (i.e. step numbers are different)

But hoping you can review the changes I've made here first and then once it looks good to you I can copy those changes to the remainder of files.